source/use_case_ndvi_investigate.rst

```diff
@@ -0,0 +1,146 @@
+
+Normal Flow
+===========
+
+.. Detailed description of the user actions and system responses that will take place during
+   execution of the Use Case under normal, expected conditions. This dialog sequence will ultimately
+   lead to accomplishing the goal stated in the Use Case name and description.
+
+* [*External system*] The advanced user defines time and region of interest for the NDVI timeseries
+* NDVI Tool searches the *S-ENDA find* system via the machine-machine API for the data of interest
+* NDVI Tool retrieves use and discovery metadata from *S-ENDA find*
+* NDVI Tool accesses the *S-ENDA access* system via OPeNDAP using the http address provided in the
+  metadata
+* NDVI Tool extracts spatial subsets of the found Sentinel-2 data
+* Subsets of the Sentinel-2 data are streamed to the NDVI Tool
+* [*External system*] The NDVI Tool creates an NDVI timeseries
+* [*External system*] The advanced user analyses and interprets the 3D NDVI dataset
+
+Alternative Flows
+=================
+
+.. Other, legitimate usage scenarios that can take place within this Use Case.
+
+None
+
+Exceptions
+==========
+
+.. Anticipated error conditions that could occur during execution of the Use Case, and how the
+   system is to respond to those conditions, or the Use Case execution fails for some reason.
+
+* OPeNDAP access could fail
+
+  * The system must offer an opportunity to download full files as a replacement for the OPeNDAP stream
+
+* The search API could fail(?)
+
+Includes
+========
+
+.. Other Use Cases that are included (“called”) by this Use Case (common functionality appearing in
+   multiple Use Cases can be described in a separate Use Case included by the ones that need that
+   common functionality).
+
+None
+
+Notes and Issues
+================
+
+.. Additional comments about this Use Case and any remaining open issues that must be resolved. (It
+   is useful to Identify who will resolve each such issue and by what date.)
+
+* We need to make a simple NDVI tool which calculates the NDVI from two bands in Sentinel-2
+
+Relevant software
+=================
+
+* Python netCDF4 to access the OPeNDAP stream
+* `Nansat <https://github.com/nansencenter/nansat>`_ could be used as basis for the NDVI tool and to
+  access the Sentinel-2 data stream or any downloaded netCDF-CF files
+
+Relevant datasets
+=================
+
+* NBS Sentinel-2
+
+UML diagram
+===========
+
+.. UML diagram, example;
+
+.. uml::
+
+   @startuml NDVI context diagram
+   !includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Context.puml
+
+   LAYOUT_LEFT_RIGHT
+
+   Boundary(users, "Users") {
+      Person(professional, "Advanced user")
+      System_Ext(ndvi_tool, "NDVI Tool")
+   }
+
+   System(senda_find, "S-ENDA Find")
+   System(senda_access, "S-ENDA Access")
+
+   Rel(professional, ndvi_tool, "Defines time and region of interest", "GUI / text based UI")
```

Comment:
Should it be some relations describing the results from senda_find and senda_access back to the Boundary users?

Reply:
I guess so. Not sure what we would need but will start with validation, usage statistic, and user feedback as suggested in the main architecture

Reply:
hmm - I understood wrongly... Thought it was about the ndvi monitoring use case. I will anyway update that.

Reply:
Regarding information flow from *S-ENDA find* and *S-ENDA access* to users, the users should be the active part, i.e., pulling information from *find* and *access*. These systems are just passive (meta)data providers as far as I understand. So the answer would then be "no"...